transmission_interface/include/transmission_interface/differential_transmission.h

```diff
@@ -123,7 +123,8 @@ class DifferentialTransmission : public Transmission
    * \param joint_offset       Joint position offset used in the position mappings.
    * \pre Nonzero actuator and joint reduction values.
    */
-  DifferentialTransmission(const std::vector<double>& actuator_reduction,
+  DifferentialTransmission(const bool &ignore_transmission_for_absolute_encoders,
```

Comment:
Why is this bool passed as const ref? Should be pass by val.

Reply:
a `const` val is has a tiny tiny caveat that it doesn't even accidentally allow you to change the value received as an argument, I settled for that.

Reply:
Fair enough, though per the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Use_of_const), `const` should not be used on parameters passed by value in function _declarations_, as it is meaningless. Using it in _definitions_ is neither encouraged nor discouraged, but left to the team/project's discretion, as they are equivalent to local const variables.

I agree with you that we should move toward adding `const` for local vars and, equivalently, function parameters passed by value in _definitions_, but not declarations. 

A final note: Adding `const` here is inconsistent with the style of the rest of the project, but since one of the topics in #403 is adding `const` where appropriate, in time the rest of the project will be brought in line with this convention :)